lightning/src/chain/onchaintx.rs

```diff
@@ -286,9 +150,9 @@ pub struct OnchainTxHandler<ChannelSigner: Sign> {
 	// us and is immutable until all outpoint of the claimable set are post-anti-reorg-delay solved.
 	// Entry is cache of elements need to generate a bumped claiming transaction (see ClaimTxBumpMaterial)
 	#[cfg(test)] // Used in functional_test to verify sanitization
-	pub pending_claim_requests: HashMap<Txid, ClaimTxBumpMaterial>,
+	pub pending_claim_requests: HashMap<Txid, PackageTemplate>,
```

Comment:
rustc doesn't like this - even though the whole module is only `pub(crate)` the fact that this includes a `pub(crate)` thing makes it sad. We can just make this `pub(crate)` instead.